safe/gui/tools/help/welcome_message.py

```diff
@@ -70,9 +70,23 @@ def content():
     :rtype: safe.messaging.message.Message
     """
     message = m.Message()
-    message.add(m.Paragraph(tr('Lorem ipsum dolot.')))
     message.add(m.Paragraph(tr(
-        'Tim, please add here your magnificent welcome message for our lovely '
-        'InaSAFE users...')))
+        'Welcome to InaSAFE. Before you begin using InaSAFE, we would '
+        'like to encourage you to review some options that will by InaSAFE. '
+        'On the "Organisation Profile" tab you can set the name of your '
+        'organisation, your organisation logo and other options related to '
+        'your organisation.'
+    )))
+    message.add(m.Paragraph(tr(
+        'The "parameters" tab is used to determine '
+        'whether people are affected and the displacement rates that apply '
+        'to people exposed to different types of hazards and at different '
```

Comment:
While I'm translating in French, I noticed: I think it's "people affected", not "people exposed". Small note on terminology ;-)